tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -451,10 +457,7 @@ TRTEngineOp::EngineCtxPair& TRTEngineOp::GetEngine(int batch_size,
     TrtUniquePtrType<IRuntime> infer(nvinfer1::createInferRuntime(logger));
 #if NV_TENSORRT_MAJOR > 3
     auto allocator = GetAllocator(ctx);
-    if (allocator == nullptr) {
-      // GetAllocator already set the Status.
-      return null_pair;
-    }
+    if (allocator == nullptr) return null_pair;
```

Comment:
please use braces even if it is a single instruction. Same for below

Reply:
Done.